board_rules.md

```diff
@@ -0,0 +1,91 @@
+# Board rules 
+
+The description of the team workflow for the board.
+
+## Description
+
+The board consists of 5 columns:
+
+- `Common backlog`- includes all tasks that should be done.
+- `Sprint backlog` - includes the list of tasks for the current sprint.
+- `In Progress` - the list of tasks that the team works on.
+- `In Review` - the list of tasks that are under review process.
+- `Done` - the list of tasks that successfully merged to main branch.
```

Comment:
What if we have done the task that does not require writing code? Will it be "merged to main branch"?

What if we have realized that some task is useless and close them. Will it be "merged to main branch"?

Reply:
Thanks, fixed it.